cmd/podman/images/push.go

```diff
@@ -108,6 +108,8 @@ func pushFlags(cmd *cobra.Command) {
 
 	flags.BoolVar(&pushOptions.TLSVerifyCLI, "tls-verify", true, "Require HTTPS and verify certificates when contacting registries")
 
+	flags.StringVar(&pushOptions.CompressionFormat, "compression-format", "", "compression format to use")
```

Comment:
Nit, remove line between this and previous line.

Reply:
Should we set default, should it be added to containers.conf?

Reply:
the default is gzip

Reply:
yes, I think this should be configurable from containers.conf

Reply:
I've opened a PR here: https://github.com/containers/common/pull/845